src/sentry/models/groupassignee.py

```diff
@@ -63,7 +64,8 @@ def assign(self, group, assigned_to, acting_user=None):
                 type=Activity.ASSIGNED,
                 user=acting_user,
                 data={
-                    'assignee': six.text_type(assigned_to.id),
+                    # todo(maxbittker) use actor_ids here?
+                    'assignee': assigned_to.id,
```

Comment:
Why are you changing this to an integer?

Reply:
This is going to break below where you check:

`six.text_type(activity.user_id) == data['assignee']`

Now becuase it's not a string, it's an integer.

Reply:
good catch